src/app/pages/details/details.hbs

```diff
@@ -93,16 +87,7 @@
                 {{/each}}
```

Comment:
I noticed you removed this `{{#each bookInfo.instructorResources}}`. Is this `{{/each}}` still needed?